.gitignore

```diff
@@ -3,6 +3,7 @@ build/
 dist/
 docs/_build/
 docs/docblocks/
+docs/images/
```

Comment:
We don't want to ignore this folder, otherwise the database diagram is missing in documentation on readthedocs.